src/components/Header/styles.ts

```diff
@@ -63,3 +69,38 @@ export const HeaderRightSide = styled.div`
   justify-self: end;
   align-self: center;
 `;
+
+export const ReturnRoute = styled.div<IReturnRoute>`
+  font-size: 20px;
+  white-space: nowrap;
+  margin-left: 50px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: white;
+  cursor: pointer;
+  pointer-events: auto;
+
+  .country {
+    font-weight: ${({ selectedLayer }) => (selectedLayer === 'country' ? 'bold' : 'normal')};
+  }
+
+  .state {
+    font-weight: ${({ selectedLayer }) => (selectedLayer === 'state' ? 'bold' : 'normal')};
+  }
+
+  .district {
+    font-weight: ${({ selectedLayer }) => (selectedLayer === 'district' ? 'bold' : 'normal')};
+  }
+`;
+
+export const ReturnRouteButton = styled.div`
+  border-radius: 50%;
+  width: 20px;
+  height: 20px;
+  background-color: white;
+  margin-right: 10px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
```

Comment:
I think this might me duplicated.

Reply:
@enzoggqs

Reply:
I didnt understant this

Reply:
It seen that you copy from here https://github.com/gpmc-lab-ufrgs/atlas/pull/41/files#:~:text=%7D-,export%20const%20ReturnRoute%20%3D%20styled.div%3CIReturnRoute%3E%60,-Write